client/my-sites/plugins/plugin-ratings/index.jsx

```diff
@@ -1,145 +1,50 @@
-import { ProgressBar } from '@automattic/components';
 import { formatNumber } from '@automattic/number-formatters';
-import { localize, getLocaleSlug } from 'i18n-calypso';
+import { localize } from 'i18n-calypso';
 import PropTypes from 'prop-types';
 import { Component } from 'react';
-import Rating from 'calypso/components/rating';
 import { gaRecordEvent } from 'calypso/lib/analytics/ga';
 
-import './style.scss';
-
-const ratingTiers = [ 5, 4, 3, 2, 1 ];
-
 class PluginRatings extends Component {
 	static propTypes = {
 		rating: PropTypes.number,
 		ratings: PropTypes?.oneOfType( [ PropTypes.object, PropTypes.array ] ),
-		downloaded: PropTypes?.number,
```

Comment:
Should we also remove the other unused props, too

Reply:
Nice catch! Will do.